pkg/smokescreen/smokescreen.go

```diff
@@ -351,13 +369,14 @@ func configureTransport(tr *http.Transport, cfg *Config) {
 
 func newContext(cfg *Config, proxyType string, req *http.Request) *smokescreenContext {
 	start := time.Now()
+
 	logger := cfg.Log.WithFields(logrus.Fields{
-		"id":             xid.New().String(),
-		"proxy_type":     proxyType,
-		"requested_host": req.Host,
-		"source_addr":    req.RemoteAddr,
-		"start_time":     start.UTC(),
-		"trace_id":       req.Header.Get(traceHeader),
+		"id":                  xid.New().String(),
+		"proxy_type":          proxyType,
+		"requested_host":      req.Host,
+		"start_time":          start.UTC(),
+		"trace_id":            req.Header.Get(traceHeader),
+		"inbound_remote_addr": req.RemoteAddr,
```

Comment:
Nit: Could we keep these field names in order?

Reply:
Reordered in eeaa285d29b702174b6eca5426843fd48727396e.